pkg/log/logger.go

```diff
@@ -96,6 +98,10 @@ func (l logger) Errorf(format string, args ...interface{}) {
 	l.entry.Errorf(format, args...)
 }
 
+func (l logger) Printf(format string, args ...interface{}) {
```

Comment:
this is to fulfill the `Logger` interface in the new elasticsearch lib, https://github.com/olivere/elastic/wiki/Logging